README.md

```diff
@@ -208,6 +210,80 @@ Also, there's a community of support from a whole team of contributors, whom you
+
+### Accepting PR
+
+#### Squash commits
+We use the __Squash and Merge__ option when merging the PR.
+This will keep the commit history clean.
+
+![GIF showing Squash functionality](https://files.gitter.im/FreeCodeCamp/Contributors/56MQ/9cb8db153d7bb1b3576cd1ffc207e39d.gif)
+
+#### PR title
+Currently under discussion, see [here](https://github.com/freeCodeCamp/guides/issues/1853).
+
+- use meaningful titles
+- avoid `Updating index.md`
+
+In example: if you create a _Variables_ article inside the _Python_ directory,  
+the pull request title should be `Python: added "Variables" article`.  
+
+**Format**: `{Parent category}: added "{Article's title}" article`
+
+
+### Closing PR
+
+We close a pull request:
+- if there is zero/little effort in it (e.g: copy pasting from another source like Wikipedia)
+- if there is copied text from a copyrighted source (see also https://github.com/freeCodeCamp/guides/issues/2503)
+- if it does not respect the [Article style guide](https://github.com/freeCodeCamp/guides#article-style-guide)
+- if it does not respect the [Academic Honesty policy](https://www.freecodecamp.org/academic-honesty)
+- if it is stale (if a change is requested and there weren't any activity for about 2 weeks)
+
+Remember that a PR can always be reopened; merge can be reverted.
+
+#### Template text for closing PR
+> Thank you for your pull request. Please read this style guide: https://github.com/freeCodeCamp/guides#article-style-guide
+I am closing this pull request for now. Please let me know if you have any questions.
+
+
+### Conflicting and duplicate PRs
+To find duplicates PRs: 
+
+1. Sort PR from the oldest
+1. Search for PRs with similar content
+1. Merge from the oldest to the newest
+
+There will be probably merge conflicts; if it is the case, notify the contributor with this template text:
+  
+#### Template text
+> We apologize for the inconvenience; however, while your PR was in the review queue someone else proposed the same file and their contribution was merged. As a result we need to resolve merge conflicts in order to merge your changes. If you're unsure about this process feel free to reach out in the contributor Gitter channel or comment below. We recommend you review the existing file and propose how you could incorporate your own ideas while maintaining the other contributors work. We will be closing this PR for now, but if you still want to see your changes added let us know and we can open it for additional commits.
+Thank you for contributing! Keep up the awesome work 
+
+### Requesting changes
+
+If a pull request is not perfect we can:
+- request changes to the contributor and adding the *change requested* label (see below)
+- fix minor issues by ourselves
+
+#### Adding Labels
+- **content** is for pull requests that modify the content of articles on the guide (e.g.: new articles or updating articles)
```

Comment:
Another nit: "change requested" => "changes requested" (plural 😜 - same comment for the other two examples below)